packages/react-core/src/components/ContextSelector/ContextSelectorToggle.tsx

```diff
@@ -32,7 +34,7 @@ export class ContextSelectorToggle extends React.Component<ContextSelectorToggle
   static displayName = 'ContextSelectorToggle';
   static defaultProps: PickOptional<ContextSelectorToggleProps> = {
     className: '',
-    toggleText: '',
+    toggleText: null as React.ReactNode,
```

Comment:
Also @nicolethoen I don't think we will need this change, however would love to know your opinion on this 👀

Reply:
I agree that it's likely unnecessary - an empty string is a valid value for a React.ReactNode

Reply:
Oki Doki, updated the PR with the changes 🚀